adding pagination themeDocs definitions (#18)
halocline/grommet: src/js/components/Pagination/README.md

```
## Theme
  
**pagination.button.active.background.color**

Background color when the button is active. Expects `string | { dark: string, light: string }`.

Defaults to

```
active-background
```

**pagination.button.color**

The color of the text label. Expects `string | { dark: string, light: string }`.

Defaults to

```
text-strong
```

**pagination.button.hover.background.color**

> Background color when the button when hovered. Expects `string | { dark: string, light: string }`.

Reviewer: ```suggestion
Background color of the button when hovered. Expects `string | { dark: string, light: string }`.
```

Author: Done.

Defaults to

```
background-contrast
```

**pagination.button.hover.color**

The color of the text label when hovered. Expects `string | { dark: string, light: string }`.

Defaults to

```
undefined
```

**pagination.container**

> Any valid Box property affecting the Box 

Reviewer: ```suggestion
Any valid Box props for the Box wrapping the pagination controls.
```
Aligning closer to other grommet docs

Author: Done.

    wrapping the pagination controls. Expects `object`.

Defaults to

```
undefined
```

**pagination.container.extend**

Any additional style for the Box wrapping 
    the pagination controls. Expects `string | (props) => {}`.

Defaults to

```
undefined
```

**pagination.control.extend**

> Any additional style for each pagination control. Expects `string | (props) => {}`.

Reviewer: ```suggestion
Any additional style for each control. Expects `string | (props) => {}`.
```
Seems that you leaned towards not using `pagination` later on

Author: Done

Defaults to

```
undefined
```

**pagination.control.pad**

> Padding around each pagination control's label. Expects `string | object`.

Reviewer: ```suggestion
Padding around each control's label. Expects `string | object`.
```

Author: Done.

Defaults to

```
4px
```

**pagination.control.size.small.border.radius**

Rounding of the corners for each control. Expects `string`.

Defaults to

```
3px
```

**pagination.control.size.small.border.width**

Border thickness for each control. Expects `string`.

Defaults to

```
2px
```

**pagination.control.size.small.font.size**

The font size of each control's label. Expects `string`.

Defaults to

```
14px
```

**pagination.control.size.small.font.height**

The line-height of each control's label. Expects `string`.

Defaults to

```
20px
```

**pagination.control.size.small.height**

> The height for each pagination control. Expects `string`.

Reviewer: ```suggestion
The height for each control. Expects `string`.
```

Author: Done.

Defaults to

```
30px
```

**pagination.control.size.small.width**

> The minimum width for each pagination control. 

Reviewer: ```suggestion
The minimum width for each control. 
```

Author: Done.

    Width will scale up fitting the control's label. Expects `string`.

Defaults to

```
30px
```

**pagination.control.size.medium.border.radius**

Rounding of the corners for each control. Expects `string`.

Defaults to

```
4px
```

**pagination.control.size.medium.border.width**

> Border thickness for each control. Expects `string`.

Reviewer: Should this specify "when pagination is size medium"?

Reviewer: Do we need to repeat all the theme structures for each size -- is there any way to consolidate that?

Author: I wondered the same. I had looked at what we do on button, heading, etc. where sizes are used and the docs seems to be exhaustive. 

In this use case, I think we are trying to communicate preciseness and have an accurate inventory of the available properties (as opposed to documentation trying to convey a concept). This also allows for deep-links to be shared as well.

That said, I am open to any suggestions you have.

Reviewer: I am comfortable leaving it as is then, especially if we do similarly in other areas.

Defaults to

```
22px
```

**pagination.control.size.large.font.height**

The line-height of each control's label. Expects `string`.

Defaults to

```
28px
```

**pagination.control.size.large.height**

The height for each pagination control. Expects `string`.

Defaults to

```
48px
```

**pagination.control.size.large.width**

The minimum width for each pagination control. 
    Width will scale up fitting the control's label. Expects `string`.

Defaults to

```
48px
```

**pagination.controls.align**

How the pagination controls should be aligned 
    within the containing Box. Expects `string`.

Defaults to

```
center
```

**pagination.controls.direction**

Direction in which the containing Box should 
    display the pagination controls. Expects `string`.

Defaults to

```
row
```

**pagination.controls.gap**

Amount of gap spacing between each control. Expects `string`.

Defaults to

```
xxsmall
```

**pagination.controls.margin**

Amount of margin surrounding the controls. Expects `string`.

Defaults to

```
none
```

**pagination.controls.pad**

Amount of pad surrounding the controls. Expects `string`.

Defaults to

```
none
```

**pagination.icons.color**

> Color to fill the controls' icons. Expects `string | { dark: string, light: string }`.

Reviewer: ```suggestion
The color to use for the icon. Expects `string | { dark: string, light: string }`.
```
To align closer to other doc wording

Author: Done.
